### DISCUSSION TITLE
Gears and power

### DISCUSSION THREAD

**Discussion by eskimos20** [cagnulein/qdomyos-zwift#2925] [Q&A] [answered]
Hi! 

I read another topic about gears and ratios. I´ve change my gearing to have a smaller step between gears.. not sure if it made any difference. Today I did a 1.5h ride and used gear 3 most of the time. For me that was about 190w @75rpm. When changing gear to 4 I had an increase of watts to about 245w @75rpm, for me I think this is to much. 

1. Can I adjust any value so that the trainer resistance will be slightly less between gears ? 
2. Are there only 13 gears compared to Zwift 24 ?

/Eskimos

**Comment by cagnulein** [comment]
You can change all the aspects of the gears under bike options, wahoo
options! You can set one of the preset with 24 gears or even customize it!

Let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 21 dic 2024 alle 18:40 eskimos20 ***@***.***>
ha scritto:

**Reply by eskimos20** [reply]
Ok. Ive set the following as pictures show and I see the gears showing correct values in the qz app but the resistance does not change. The wattage is the same. About 190w @80rpm on 3 gear and about 225w @80rpm on 4 gear. I must be doin something wrong 😊
![20241222_173323](https://github.com/user-attachments/assets/0715feac-a4ba-4bf9-baa2-bbf3f68d86ee)
![20241222_173311](https://github.com/user-attachments/assets/b5c11453-e64d-45a2-8247-e8f907299ac5)

**Reply by cagnulein** [reply]
Attach a debug log from a session so I can check your setup https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Reply by MeckGeiwa** [reply]
Does this virtual gear Table only work with wahoo Trainers or also with other trainers like Tacx?

**Reply by cagnulein** [reply]
@MeckGeiwa For now the wahoo table don't work on tacx but I already did a PR for expanding also on tacx but I need android tester.are you on android?

**Reply by MeckGeiwa** [reply]
Yes, i have the android app and use a taxc neo 3m Trainer.

**Reply by cagnulein** [reply]
@MeckGeiwa

This is the pull request https://github.com/cagnulein/qdomyos-zwift/pull/2734

And this is the test app, comments on the pr for the feedback https://github.com/cagnulein/qdomyos-zwift/actions/runs/12315201304/artifacts/2317096895

**Reply by MeckGeiwa** [reply]
Thank you! I will test later and give feedback.

**Reply by cagnulein** [reply]
@MeckGeiwa it's very experimental, so i don't know if it will work, i'm looking for testers for this reason :)

**Reply by MeckGeiwa** [reply]
I did a very short test and it seams to work. I testet with 3 virtual gears. My chainring/cog was on 36/17 und my 3 virtual gears were:

36/17 - no chance in the resistance 
36/11 - the resistance got higher
36/28 - the resistance got lower

So it works for me. I had to activate FTMS Bike - otherwise my Tacx Neo 3m wouldn't connect to the QZ app.

**Reply by cagnulein** [reply]
Ok @MeckGeiwa so in this way you are not using the wahoo options, because ftms bike use the standard qz gearing 

You can use the gear gain to decrease the strength of the gears, like 0.5 for example 

If it's good for you, you can simply use this on the play store beta version https://play.google.com/apps/testing/org.cagnulen.qdomyoszwift

**Comment by eskimos20** [comment]
I also have a tacx neo 2 so that makes sence why it is not working at all.

Den ons 25 dec. 2024 09:13MeckGeiwa ***@***.***> skrev:

**Reply by cagnulein** [reply]
@eskimos20virtual gearing it works with your trainer already, it doesn't handle the gear customizing in the wahoo table. But it already works!

**Reply by eskimos20** [reply]
For the Neo 2 where do I need to change for the resistance for every gear to be less big, is it changing the value under:

Bike Options / Gears Gain from 1 --> 0.5 ?

**Reply by cagnulein** [reply, marked as answer]
yes exactly @eskimos20

**Reply by eskimos20** [reply]
Perfect! I will try this today! :)

**Comment by eskimos20** [comment]
Ok! I will check on that later. The winter flu came for a visit 😊

Den ons 25 dec. 2024 09:24Roberto Viola ***@***.***> skrev:

**Reply by cagnulein** [reply]
Me too :(
